src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -379,13 +380,12 @@ public void run() {
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.pipeLarge, Materials.Enderium, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Osmium, 8L), GT_Utility.getIntegratedCircuit(4)}, GT_Values.NF, ItemList.Casing_Tank_8.get(1L), 100, 16);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.pipeLarge, Materials.Naquadah, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Neutronium, 8L), GT_Utility.getIntegratedCircuit(4)}, GT_Values.NF, ItemList.Casing_Tank_9.get(1L), 100, 16);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.pipeLarge, Materials.MysteriousCrystal, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Bedrockium, 8L), GT_Utility.getIntegratedCircuit(4)}, GT_Values.NF, ItemList.Casing_Tank_10.get(1L), 100, 16);
+
         GT_Values.RA.addAssemblerRecipe(new ItemStack(Items.diamond, 1, 0), ItemList.Circuit_Basic.get(4L), CustomItemList.TwilightCrystal.get(1L), 600, 16);
         GT_Values.RA.addAssemblerRecipe(new ItemStack(Items.diamond, 1, 0), GT_ModHandler.getModItem("IC2", "itemPartCircuit", 4L, 0), CustomItemList.TwilightCrystal.get(1L), 600, 16);
 
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Gold, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Diamond, 1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Glass.getMolten(72L), GT_ModHandler.getModItem("irontank", "goldDiamondUpgrade", 1L, 0), 600, 120);
 
-        GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.pipeLarge, Materials.MysteriousCrystal, 1L), CustomItemList.BedrockiumPlate.get(8L), ItemList.Casing_Tank_10.get(1L), 100, 16);
-
```

Comment:
why you remove this recipe ?

Reply:
Did you see the part where I said I removed a duplicate?